test/unit/module/registry/test_aws_call_registry.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+import cfnlint.core
+import cfnlint.registry
+from test.testlib.testcase import BaseTestCase
+
+from mock import patch
+
+LOGGER = logging.getLogger('cfnlint')
+
+
+class MyTestAWSCallRegistry(BaseTestCase):
```

Comment:
You don't need prefix `My` in unit test. Also test cases should cover failure scenarios as well.